### ISSUE TITLE
"angleButtons": "" still showing up dmaker-fan-p9

### ISSUE THREAD

**Issue by DJay-X** [merdok/homebridge-xiaomi-fan#57]
Today I received my dmaker-fan-p9. And of course your plugin works great again @merdok Thanks as always. 👍 

However, I don't want the the angle buttons to show up, so I don't have them in the config.json

```
{
                    "name": "Turmventilator",
                    "ip": "xxx.xxx.xxx.xx",
                    "token": "00000000000000000000000000000000",
                    "model": "dmaker.fan.p9",
                    "pollingInterval": 10,
                    "moveControl": false,
                    "buzzerControl": true,
                    "ledControl": true,
                    "naturalModeButton": true,
                    "shutdownTimer": false,
                    "fanLevelControl": false
                }
```
But 5 buttons showing up. I added "angleButtons": "" but they are still there after a reboot.
"angleButtons": false - what of course it not right - removed them, but with an error in the log.
[xiaomifan] [Turmventilator] The angle buttons service needs to be defined as an array! Please correct your config.json if you want to use the service.

Any help is much appreciated how to get rid of this extra buttons. :)

**Comment by merdok** [comment]
The idea is that right now if the fan supports predefined angle buttons, I read them out and display them if the user did not specify their own values. In order to remove the angle buttons completely set the value of the property as an empty array []. That should remove them😉

**Comment by DJay-X** [comment]
Ah.. that wasn't clear to my reading the description of Fan Configuration fields. Thanks.

This should be correct? But after a restart of homebridge the buttons are still there. 

```
{
                    "name": "Turmventilator",
                    "ip": "xxx.xxx.xxx.xx",
                    "token": "00000000000000000000000000000000",
                    "model": "dmaker.fan.p9",
                    "pollingInterval": 10,
                    "moveControl": false,
                    "buzzerControl": true,
                    "ledControl": true,
                    "naturalModeButton": true,
                    "shutdownTimer": false,
                    "fanLevelControl": false,
                    "angleButtons": []
                }
```

**Comment by merdok** [comment]
Yeah, it should be correct like that. Well then it seems that this is a bug, I will check it out and fix it.

**Comment by merdok** [comment]
Should be fixed in version 1.4.3. I also updated the README to clarify that more! Please let me know if the issue is fixed!

**Comment by DJay-X** [comment]
Thanks a lot. That did work instantly. 👍
